app/controllers/proposal/collection_draft_proposals_controller.rb

```diff
@@ -226,6 +230,44 @@ def submitter_from_resource
       { name: "#{proposal_urs_user['first_name']} #{proposal_urs_user['last_name']}", email: proposal_urs_user['email_address'] } unless proposal_urs_user.blank?
     end
 
+    # Get urs users and get emails from urs for approvers
+    # Returns array of hashes for each user
+    def get_approver_emails
```

Comment:
This is a nice idea, getting the emails from the ACLs. We should make sure we are getting all the right emails -- in SIT and UAT we check 'SCIOPSTEST' for the Draft MMT ACLs as well.